recipes-extended/realsense-camera/realsense-camera/0001-librealsense-Fix-link-when-the-system-library-is-use.patch

```diff
@@ -0,0 +1,30 @@
+From fb74e5eec2368f20c4b43cf9224519b29aee35ad Mon Sep 17 00:00:00 2001
+From: Murilo Belluzzo <murilo.belluzzo@intel.com>
+Date: Wed, 5 Apr 2017 17:06:44 -0700
+Subject: [PATCH 3/3] librealsense: Fix link when the system library is used
+
+Otherwise librealsense won't be listed as a dependency for the nodelet
+that will fail to load due to undefined symbols.
+
```

Comment:
Is there an example on how the status should be stated on the message? It is merged upstream but is not on any release yet.